meilies/src/stream/stream.rs

```diff
@@ -7,51 +7,58 @@ use crate::resp::{RespValue, FromResp, RespStringConvertError};
 use crate::stream::{StreamName, StreamNameError};
 
 #[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub enum StartReadFrom {
+pub enum ReadPosition {
     EventNumber(u64),
     End,
 }
```

Comment:
I am asking myself, wouldn't it be better to unify these bounds in one type `ReadRange`?

```rust
enum ReadRange {
   ReadFromUntil(u64, u64),
   ReadFrom(u64),
   ReadToEnd,
}
```